README.md

```diff
@@ -32,76 +30,50 @@ TrueBlocks is a collection of libraries, tools, and applications that improve ac
 
 3) TrueBlocks enhances the Ethereum RPC interfaces. For example, you may query blocks and transactions by date, by block range, by hashes or any combination. Furthermore, two additional endpoints are provided for extracting (`export`) and listing (`list`) historical transactions per address.
 
-## Prerequisites
+## Quick Install
 
----
-
-**Golang**:
+TrueBlocks runs on Linux and Mac. There is no official Windows support.
+Some users have had success using WSL─you're on your own!
 
-For all operating systems, please follow [these instructions](https://golang.org/doc/install) to install `golang` on your system.
+These instructions assume you can navigate directories with the command line
+and edit configuration files.
+If you need help with a step, see the [installation's troubleshooting section](https://trueblocks.io/docs/prologue/installing-trueblocks/#troubleshooting).
 
-**For Linux**:
+0. Install dependencies
+    - &#9745; [Install Go, version 1.12x or later](https://golang.org/doc/install).
+    - &#9745; Install the other dependencies with your command line: `build-essential` `git` `cmake` `python` `python-dev` `libcurl3-dev` `clang-format` `jq`.
```

Comment:
My thinking is that, this README is for people who have a good sense of how to install packages. But I want to catch everything, so I added the widest possible dependencies.

This list includes everything that someone on a mac needs, and a little more. Actually, it's more even than what's needed on Ubuntu, which I believe typically comes preinstalled with git.

 Also, on the troubleshooting, I added specific examples for installing packages with Ubuntu and Mac.
 https://trueblocks.io/docs/prologue/installing-trueblocks/#how-do-i-install-packages-on-the-command-line

Reply:
Sorry to be pendantic about this, but I'd like to try to put this to bed once and for all.

I would call myself very familiar with installing packages and using the command line. If I saw this line (which basically says -- go install a bunch of stuff), I would be very annoyed. I want copy and paste instructions for my operating system -- but -- I'm willing to do just Mac and Linux. (We don't need to do Windows, nor any other Linux version other than Ubuntu -- if someone is using non-Ubuntu, they will figure it out).

I see your trouble shooting section, which is good, but perhaps we can at least link to that directly here with something like "For operating specific command, see here."

The thing I want to avoid is annoying people like me who know how to do thing but are uber-lazy and want copy and paste commands that work without any thought on my part.